forestserviceprototype/static_assets/additional.css

```diff
@@ -0,0 +1,11 @@
+.help-text {
+    font-style: italic;
+    font-weight: normal;
+}
+.permit_hide {
+  display: none;
+}
+
+input[disabled], fieldset[disabled] input, select[disabled] {
+    background-color: #eee;
```

Comment:
If we were using SASS I'd recommend this be a variable, but for the sake of speedy development it makes sense to run with this.